VogtPayroll6/Payroll.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace VogtPayroll6
+{
+    class Payroll
+    {
+        public decimal grossPaytotal = 0.00m;
+
+        public void DisplayPayroll(Employee[] empArray) {
+
+            foreach (var employee in empArray)
+            {
```

Comment:
Your payroll class isn't doing much. This should be managing the employees. Your console reader should read in the employees. Then you should be passing them to the payroll to store the list. Every time you add an employee, you update payroll. That would fix your issue with the previous employee list not being saved.

All of the employee management/interaction should be here. Including PrintAllEmployeeInfo().